wpeverest/everest-forms
# Fix - Critical issue when the publishable key is empty in live mode for recurring payment

### All Submissions:






### Changes proposed in this Pull Request:



This pull request addresses a critical issue in the **Everest Forms Stripe** integration that arises when **test publishable key and test secret key** are set without enabling test mode. Specifically, when users attempt to **enable recurring payments** from the form settings under these conditions, it leads to an error that affects the functionality of the integration.

Closes # .

### How to test the changes in this Pull Request:

1. Enable the **Everest Form Stripe Addons**
2. Go to **Everest Forms Settings > Payments >  Stripe Settings**
3. Enter the **Test Publishable Key, Test Secret Key** and disable **Enable test Mode**
4. Now go to **Form settings > Payments**
5. **Enable Stripe Recurring Payment** and click on **Save** button
6. Now try to drag and drop the credit card field.
7. A pop up must appear to show the message and button must redirect to the setup page.

[PR Dependency](https://github.com/wpeverest/everest-forms-pro/pull/971)

### Types of changes:




### Other information:




### Changelog entry

> Fix - Critical issue when the publishable key is empty in live mode for recurring payment.
